xbmc/cores/DataCacheCore.h

```diff
@@ -117,4 +122,12 @@ class CDataCacheCore
     float m_tempo;
     float m_speed;
   } m_stateInfo;
+
+  struct STimeInfo
+  {
+    time_t m_startTime;
+    int64_t m_time;
```

Comment:
Why are these allowed to be negative? I noticed that a lot of the checks for negative have been dropped. Is this intended to allow negative times?